<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miniRoomScreen</class>
 <widget class="QMainWindow" name="miniRoomScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ミニ画面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <widget class="QLabel" name="insideRoomLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../files.qrc">:/image/images/backgrounds/room_day.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="hoverFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="smaphoButton">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>120</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../files.qrc">
       <normaloff>:/image/images/icons/pink_smapho.png</normaloff>:/image/images/icons/pink_smapho.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="chromeButton">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>120</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../files.qrc">
       <normaloff>:/image/images/icons/blue_chrome.png</normaloff>:/image/images/icons/blue_chrome.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="maximizeButton">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>120</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../files.qrc">
       <normaloff>:/image/images/icons/maximize.png</normaloff>:/image/images/icons/maximize.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
    </widget>
    <zorder>chromeButton</zorder>
    <zorder>smaphoButton</zorder>
    <zorder>maximizeButton</zorder>
   </widget>
   <widget class="QLabel" name="osanaLabel">
    <property name="geometry">
     <rect>
      <x>-130</x>
      <y>-7</y>
      <width>600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../files.qrc">:/image/images/character/figure_good.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>insideRoomLabel</zorder>
   <zorder>osanaLabel</zorder>
   <zorder>hoverFrame</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>smaphoButton</tabstop>
  <tabstop>chromeButton</tabstop>
  <tabstop>maximizeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../files.qrc"/>
 </resources>
 <connections/>
</ui>
